oak_runtime/src/node/http/tests.rs

```diff
@@ -30,7 +30,7 @@ struct HttpServerTester {
 }
 
 impl HttpServerTester {
-    fn new(port: u32, with_simulator_thread: bool) -> HttpServerTester {
+    fn new(port: u32, number_of_requests: i32) -> HttpServerTester {
```

Comment:
Could you document this parameter?

Reply:
Done.

Reply:
It may not make sense to repeat it, but I think it makes sense to have a comment saying where to look, if someone were looking at the documentation of this method (as opposed to reading through its implementation).